app/native/components/Collapsible.tsx

```diff
@@ -0,0 +1,45 @@
+import Ionicons from "@expo/vector-icons/Ionicons";
```

Comment:
So we're gonna use Material UI for the frontend and expo for the app, is that right?